newIDE/app/src/Profile/ProfileDetails.js

```diff
@@ -118,6 +119,13 @@ export default ({ onEditProfile, authenticatedUser }: Props) => {
               />
             </Line>
             <Line justifyContent="flex-end">
+              <RaisedButton
+                label={<Trans>Change my email</Trans>}
+                primary
```

Comment:
Let's remove primary to give it a bit less visibility compare to editing the profile